library/Zend/Code/Generator/ValueGenerator.php

```diff
@@ -239,7 +318,7 @@ public function generate()
                 );
                 foreach ($rii as $curKey => $curValue) {
                     if (!$curValue instanceof ValueGenerator) {
-                        $curValue = new self($curValue);
+                        $curValue = new self($curValue, self::TYPE_AUTO, self::OUTPUT_MULTIPLE_LINE, $this->getConstants());
```

Comment:
Also you should keep in mind of allocating memory for every array in method. That 100 classes with 100 properties would be have a 10000 arrays